images/osl-cli.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024 Apache Software Foundation (ASF)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: "packager"
+  version: 1.33.0-snapshot
+  from: "registry.access.redhat.com/ubi8/ubi-minimal:8.9-1161"
+  modules:
+    repositories:
+      - path: modules
+    install:
+      - name: com.redhat.osl.cli.packager
+
+- name: "kn-workflow-cli-artifacts"
+  version: 1.33.0-snapshot
+  description: "Red Hat OpenShift Serverless Logic 1 kn-workflow CLI artifacts"
+  from: "registry.access.redhat.com/ubi8/ubi-minimal:8.9-1161"
+  labels:
+    - name: com.redhat.component
+      value: "openshift-serverless-1-kn-workflow-cli-artifacts-rhel8-container"
+    - name: name
+      value: "openshift-serverless-1/kn-workflow-cli-artifacts-rhel8"
+    - name: version
+      value: "1.33.0-snapshot"
+    - name: summary
+      value: "Red Hat OpenShift Serverless Logic 1 kn-workflow CLI artifacts"
+    - name: description
+      value: "'kn-workflow' CLI cross platform binaries for managing Red Hat OpenShift Serverless Logic 1"
+    - name: io.k8s.display-name
+      value: "Red Hat OpenShift Serverless Logic 1 kn-workflow CLI artifacts"
+    - name: io.k8s.description
+      value: "'kn-workflow' CLI cross platform binaries for managing Red Hat OpenShift Serverless Logic 1"
+    - name: io.openshift.build.source-location
+      value: "https://github.com/apache/incubator-kie-tools/"
+    - name: maintainer
+      value: "serverless-support@redhat.com"
+  modules:
+    repositories:
+      - path: modules
+    install:
+      - name: com.redhat.osl.cli.artifacts
+  run:
+    user: 65532
```

Comment:
We need a block like this to build in OSBS:

```
packages:
  manager: microdnf
  content_sets_file: content_sets.yaml

osbs:
  configuration:
    container:
      platforms:
        only:
        - x86_64
        - aarch64
        - ppc64le
      compose:
        pulp_repos: true
  repository:
    name: containers/openshift-serverless-1-logic-kn-workflow-cli-artifacts
    branch: openshift-serverless-1.33-rhel-8
```

Note that the contents_set.yaml was removed in upstream but I will recreate it in midstream as part of SRVLOGIC-287

Reply:
ok, I will delete this comment and add it as a suggestion